### PR TITLE
Update home page and styles

### PR DESCRIPTION
**Pull request by zsotroav**
Yeah, it's two in one, but I'm kind of building on this for the rest of the website.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,29 +8,29 @@
       content="Kyuuto bot is an open-source bot written in Kotlin by the Camp Buddy Official Fan Server members."
       name="description"
     />
-    <link href="./media/css/master.min.css" rel="stylesheet" type="text/css" />
-    <script defer src="./media/scripts/main.js"></script>
+    <link href="/media/css/master.min.css" rel="stylesheet" type="text/css" />
+    <script defer src="/media/scripts/main.js"></script>
```
Comment (iamdeja) [inline index.html]: Dot prefix notation in the head, or no slash.

File: index.html [hunk 2]
```diff
@@ -8,29 +8,29 @@
...
     <script>
       document.documentElement.classList.replace("no-js", "js");
     </script>
   </head>
   <body data-theme="dark">
     <div class="header-container">
       <header class="header">
-        <a class="header__logo" href="./">Kyuuto</a>
+        <a class="header__logo" href="/">Kyuuto</a>
```
Comment (iamdeja) [inline index.html]: Here the same, empty or `./`

File: index.html [hunk 3]
```diff
@@ -62,18 +62,22 @@ <h1 class="u-text--c"><span class="t-highlight">K</span>yuuto Bot</h1>
...
               src="https://discordapp.com/widget?id=684392803131850779&theme=dark"
-              width="300"
-            ></iframe>
+              width="300">
+          </iframe>
           </div>
         </div>
       </section>
     </main>
+    <br>
```
Comment (iamdeja) [inline index.html]: What is the purpose?

File: media/css/master.scss [hunk 4]
```diff
@@ -38,6 +38,10 @@ $break-tablet--before: 767px;
   --jb-sky-blue: #05c1fd;
   --jb-green: #18d68c;
   --jb-lemon-yellow: #fcf84a;
+
+// Yuuto and CampBuddy related colors
```
Comment (iamdeja) [inline media/css/master.scss]: Move the JetBrains comment inside as well in that case.

File: media/css/master.scss [hunk 5]
```diff
@@ -233,6 +237,66 @@ h6,
   padding-top: 50px;
 }
 
+// 1.3 General buttons
+
+:root,
```
Comment (iamdeja) [inline media/css/master.scss]: No need to sleect the root element, it will only cause issues.

File: media/css/master.scss [hunk 6]
```diff
@@ -233,6 +237,66 @@ h6,
...
+  }
+  .btn.disabled,
+  .btn:disabled {
+    color:#fff;
+    background-color:#343a40;
+    border-color:#343a40
+  }
+}
+[data-theme = "dark"] {
+  .btn{
```
Comment (iamdeja) [inline media/css/master.scss]: Repeated code with light theme btn.

**Review by iamdeja — CHANGES_REQUESTED** [review]

File: index.html (hunk 3 above)
Comment (zsotroav) [reply index.html]: To have a little extra space under the page,. Without it, the Discord widget touches the bottom.

**Comment by iamdeja** [comment]
Also, run prettier on the HTML and SCSS.

File: index.html (hunk 3 above)
Comment (iamdeja) [reply index.html]: Fix it with styles

File: media/css/master.scss [hunk 7]
```diff
@@ -233,6 +237,59 @@ h6,
...
+    border-color:#f8f9fa;
+  }
+  .btn.focus,
+  .btn:focus,
+  .btn:hover {
+    color:#212529;
+    background-color:#e2e6ea;
+    border-color:#dae0e5;
+  }
+  .btn.disabled,
```
Comment (iamdeja) [inline media/css/master.scss]: Duplicate code still

**Review by iamdeja — CHANGES_REQUESTED** [review]
Don't forget to run prettier on the HTML at least.

File: index.html [hunk 8]
```diff
@@ -62,18 +62,22 @@ <h1 class="u-text--c"><span class="t-highlight">K</span>yuuto Bot</h1>
                 anyone wanting to build their own Open Source collaborative
                 project.
               </p>
+              <br>
+              <a class="btn" href="/docs/">Read the docs ></a>
```
Comment (iamdeja) [inline index.html]: Slash issue here too

File: media/css/master.scss [hunk 9]
```diff
@@ -233,6 +237,59 @@ h6,
   padding-top: 50px;
 }
 
+// 1.3 General buttons
+.btn {  
+  margin-top: 32px;
+  padding-top: 16px;
```
Comment (iamdeja) [inline media/css/master.scss]: Combine padding

File: media/css/master.scss [hunk 10]
```diff
@@ -233,6 +237,59 @@ h6,
   padding-top: 50px;
 }
 
+// 1.3 General buttons
```
Comment (iamdeja) [inline media/css/master.scss]: Can you make it 1.3 Objects, and then `.o-btn` please? The benefit of SCSS, is that you needn't to rewrite each time `.btn:selector` making for an easy addition of the `o-` prefix.

You can use the SCSS cascading with `&` for that:
```css
parent {
  // Parent styles
  &:selector,
  &:selector2,
  &:selector3 {
    // Selector styles
  }
  &.class {
    // Adjacent class styles
  }
}
```

Comment (iamdeja) [reply media/css/master.scss]: Example:
image
would be the same as styling
```css
.section--masthead h1 {
  // Styles
}
```

**Comment by iamdeja** [comment]
I'll fix what's left due to time issues.

**Review by iamdeja — COMMENTED** [review]
Merge ratification

**Review by iamdeja — COMMENTED** [review]
Is the pending review status ever disappearing?

**Pull request merged**